submissions/VBystrov/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,305 @@
+                <input type="checkbox" class="seat-checkbox" value="r5s4" />
+                <div class="seat-number">4</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r5s5" />
+                <div class="seat-number">5</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r5s6" />
+                <div class="seat-number">6</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r5s7" />
+                <div class="seat-number">7</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r5s8" />
+                <div class="seat-number">8</div>
+              </label>
+            </div>
+
+            <div class="row-count">6</div>
+            <div class="row">
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s1" />
+                <div class="seat-number">1</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s2" />
+                <div class="seat-number">2</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s3" />
+                <div class="seat-number">3</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s4" />
+                <div class="seat-number">4</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s5" />
+                <div class="seat-number">5</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s6" />
+                <div class="seat-number">6</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s7" />
+                <div class="seat-number">7</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s8" />
+                <div class="seat-number">8</div>
+              </label>
+              <label class="seat-place">
+                <input type="checkbox" class="seat-checkbox" value="r6s9" />
+                <div class="seat-number">9</div>
+              </label>
+            </div>
+          </div>
+          <div class="legend">
+            <div>
+              <p>Free</p>
+              <div class="row">
+                <label class="seat-place">
+                  <input
```

Comment:
Functionally it's just info elements, no need to use interactive elements for it.

Reply:
Input removed.